docs/mp/config/01_introduction.adoc

```diff
@@ -43,29 +81,88 @@ Step-by-step guide about using {spec-name} in your Helidon MP application.
 --
 ====
 
-== Using {spec-name} in Helidon MP applications
 
-=== Configuration
+=== Using MicroProfile Config Sources and META-INF
+
+Default configuration values can be retrieved from system properties, environment variables, files, directories, URLs or configuration files of the form `${CLASSPATH}/META-INF/microprofile-config.properties`. 
+
+The example below shows how the MicroProfile configuration file `microprofile-config.properties` can be used to modify properties such as server port or host IP. 
+
+----
+# Application properties. This is the default greeting
+app.greeting=Hello
+
+# Microprofile server properties
+server.port=8080
+server.host=0.0.0.0
+----
+
+=== Using MicroProfile Config API
+
+You can use the default MicroProfile configuration by injecting configuration properties or use `ConfigProvider.getConfig()`.
+
+----
+   org.eclipse.microprofile.config.Config config = ConfigProvider.getConfig();
+----
+
+=== Using Helidon Meta Configuration
+
+The meta configuration uses underlying implementation of Helidon config. In this case, you can either inject configuration properties or use `ConfigProvider.getConfig()`.
+Instead of directly specifying the configuration sources in your code, you can use meta configuration in a file that declares the configuration sources and their attributes. The contents of the meta-configuration file need to be in JSON, YAML, or HOCON format.
+The meta-config allows configuration of config sources and other configuration options, including retry policies and polling strategies.
+
+Example of a YAML meta configuration file: 
+
+----
+sources:
+  - type: "system-properties" <1>
+  - type: "environment-variables" <2>
+  - type: "file"
+    properties:
+      optional: true
+      path: "mp-config.yaml" <3>
+  - type: "classpath"
+    properties:
+      optional: true
+      resource: application.yaml <4> 
+    type: "classpath"
```

Comment:
missing a `-` before `type`

Reply:
Also for this specific source add
`multi-source: true`
as a new line after `- type: "classpath"